.github/workflows/merge-to-stage.js

```diff
@@ -236,7 +236,8 @@ const main = async (params) => {
     const stageToMainPR = await getStageToMainPR();
     console.log('has Stage to Main PR:', !!stageToMainPR);
     if (stageToMainPR) body = stageToMainPR.body;
-    existingPRCount = body.match(/https:\/\/github\.com\/adobecom\/milo\/pull\/\d+/g).length;
+    console.log('Current PR body:', body);
```

Comment:
Would this log actually be helpful?

Reply:
If this errors out in the future on an existing PR, we know which body was detected. Im fine to remove it though

Reply:
Less is more, removed it!
_I added it because I was unsure on how this reproduces, given that in the meantime I found out that string.match returns null rather than an empty array, the purpose of adding this has already been fulfilled. So gone with it!_